doc/quickstart/virtualization_quickstart.rst

```diff
@@ -27,8 +27,9 @@ Virtual Machine Advantages
 System Requirements
 --------------------------------------------------------------------------------
 
-* RAM: 2 GB at least. The Lubuntu system runs well with 1 GB of RAM, and you'll need to keep at least the same amount of memory for your host system. So a total of 2 GB will be needed for smooth operation. Some applications, like geoserver, needs that the virtual machine has at least 2GB of RAM. So if possible, consider providing 2 GB or more for your virtual machine.
-* Spare Hard Disk Space: The virtual disk file (vmdk) from :doc:`live.osgeo.org <../download>`  unzips to almost 10 GB. And you'll want more space to allow some room to work on the virtual machine. So 20 GB is a good recommendation.
+* RAM: 2 GB at least. The Lubuntu system runs well with 1 GB of RAM, and you'll need to keep at least the same amount of memory for your host system.
+  So a total of 2 GB will be needed for smooth operation. Some applications, like GeoServer, needs the virtual machine to have at least 2GB of RAM. So if possible, consider providing 2 GB or more for your virtual machine.
```

Comment:
From the way that you are indenting this line I deduce you want that sentence in another line (like another paragraph), if that is the case it is not working, This I generated locally:
![image](https://user-images.githubusercontent.com/5035290/95266685-47ee9000-07f9-11eb-9346-2b21321f27ba.png)
you can add a `* `  as follows.
```
* All ....

  * So ...

* Spare ...
```
Notice the blank line that is needed in the sphinx syntax to be considered a different paragraph when there is an indentation on a bullet list

Reply:
If it was not the intention just resolve the comment

Reply:
There was no intention to add a new paragraph - it was just that line went unbroken so was very hard to read in the editor (it went way off the screen).